.azure-pipelines/ultimate-pipeline.yml

```
@@ -265,6 +266,7 @@ stages:
 
     - script: tracer\build.cmd BuildProfilerHome
       displayName: Build profiler home
+      retryCountOnTaskFailure: 5
```

Same argument as above 

```suggestion
      retryCountOnTaskFailure: 1
```